rust/src/lib/route.rs

```diff
@@ -473,3 +478,64 @@ impl MergedRoutes {
         !self.route_changed_ifaces.is_empty()
     }
 }
+
+// Validating if the route destination network is valid,
+// 0.0.0.0/8 and its subnet cannot be used as the route destination network
+fn validate_route_dst(dst: &str) -> Result<(), NmstateError> {
+    if !is_ipv6_addr(dst) {
+        if dst.contains('.') {
+            if dst.contains('/') {
```

Comment:
You can do split regardless it contains `/` or not, you could save some lines of codes.